jax/_src/random.py

```diff
@@ -1828,3 +1828,46 @@ def ball(
   g = generalized_normal(k1, p, (*shape, d), dtype)
   e = exponential(k2, shape, dtype)
   return g / (((jnp.abs(g) ** p).sum(-1) + e) ** (1 / p))[..., None]
+
+
+def rayleigh(key: KeyArray,
+              scale: RealArray,
+              shape: Optional[Shape] = None,
+              dtype: DTypeLikeFloat = dtypes.float_) -> Array:
```

Comment:
Please fix indentation here.

Reply:
Thanks for your review, modification is committed